platformio.ini

```diff
@@ -511,10 +517,11 @@ build_flags = ${common.build_flags_esp8266} -D LEDPIN=12 -D IRPIN=-1 -D RLYPIN=2
 lib_deps = ${esp8266.lib_deps}
 
 [env:lolin_s2_mini]
-platform = {esp32s2.platform}
+platform = ${esp32s2.platform}
+platform_packages = ${esp32s2.platform_packages}
 board = lolin_s2_mini
 board_build.partitions = tools/WLED_ESP32_4MB_1MB_FS.csv
-build_unflags = ${common.build_unflags}
+build_unflags = ${common.build_unflags} -DARDUINO_USB_CDC_ON_BOOT=1
```

Comment:
This wrecks havoc on my Lolin S2 mini board.
Has to be 0 for it to work.

Reply:
It's an "unflag". It mean that `-DARDUINO_USB_CDC_ON_BOOT=1` will be delete from build_flags. The lolin board.json in framework 5.2.0 included this flag, so this line removes it again.